docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md

```diff
@@ -43,18 +42,18 @@ See a request body example below (10 days):
 {
     "waitingTime": 10
     }
-```    
-    
-After sending the request, the endpoint will return `code 201 (Created)` and an empty response body. 
+```
+
+After sending the request, the endpoint will return `code 201 (Created)` and an empty response body.
 
-To confirm that the new window to change seller have been applied to your account, access the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint again as a **GET** request.
+To confirm that the new seller change window has been applied to your account, access the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint again as a **GET** request.
```

Comment:
Atualizar link: [Get window to change seller](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pvt/configuration/window-to-change-seller)